### PR TITLE
Add k8s deployment yamls

### PR DESCRIPTION
**Pull request by youngbupark**
# Description

Add k8s deployment yamls

## Issue reference

n/a

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: examples/invoke-simple/deploy/invoke-caller.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      labels:
+        app: invokecaller
+      annotations:
+        dapr.io/enabled: "true"
+        dapr.io/app-id: "invoke-caller"
+        dapr.io/app-protocol: "grpc"
+    spec:
+      containers:
+      - name: invokecaller
+        image: youngp/invokesimple:latest  # EDIT HERE: Replace the image name
```
Comment (vinayada1) [inline examples/invoke-simple/deploy/invoke-caller.yaml:26]: Change image name

File: examples/invoke-simple/deploy/invoke-caller.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        command: ["python"]
+        args: ["/app/invoke-caller.py"]
+        imagePullPolicy: Always
```
Comment (vinayada1) [inline examples/invoke-simple/deploy/invoke-caller.yaml:29]: Should be Never?

Comment (youngbupark) [reply examples/invoke-simple/deploy/invoke-caller.yaml:29]: nope it will keep download new one.

**Review by vinayada1 — DISMISSED** [review]

File: examples/invoke-simple/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,10 @@
+FROM python:3.7-buster
```
Comment (mukundansundar) [inline examples/invoke-simple/Dockerfile:1]: Can the image be 3.7-slim-buster?

https://hub.docker.com/layers/python/library/python/3.7-slim-buster/images/sha256-38f7320f8b7fd0e635f3bb298a43bb47c07de5f0e2df059a2a55b7ff197dedf0?context=explore

and does pip work with a slim buster image?

File: examples/invoke-simple/README.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ This example utilizes a receiver and a caller for the OnInvoke / Invoke function
 
 > **Note:** Make sure to use the latest proto bindings
 
-## Running
+## Running in self-host mode
```
Comment (mukundansundar) [inline examples/invoke-simple/README.md:7]:
```suggestion
## Running in self hosted mode
```

File: examples/invoke-simple/Dockerfile (hunk 3 above)
Comment (youngbupark) [reply examples/invoke-simple/Dockerfile]: it works.

**Review by vinayada1 — APPROVED** [review]

**Pull request merged**